validator/validator/validator.py

```diff
@@ -238,7 +242,9 @@ def load_state(self):
         state = load(path)
         self.step = state["step"]
         self.hotkeys = state["hotkeys"]
-        self.scores = state["scores"]
+        self.metrics.baseline_averages = state["baseline_averages"]
```

Comment:
This will fail for existing validators, use `.get` with a default for things like that.